src/MDMAnimatableKeyPaths.h

```diff
@@ -180,6 +180,17 @@ FOUNDATION_EXPORT MDMAnimatableKeyPath MDMKeyPathStrokeStart NS_SWIFT_NAME(strok
  */
 FOUNDATION_EXPORT MDMAnimatableKeyPath MDMKeyPathStrokeEnd NS_SWIFT_NAME(strokeEnd);
 
+/**
+ Transform.
+
+ Equivalent UIView property: transform (2d only)
+ Equivalent CALayer property: transform (3d)
+ Expected value type: CGAffineTransform, CATransform or NSValue with either transform type.
+                      CGAffineTransform value types will be converted to CATransform.
```

Comment:
Where are CGAffineTransforms converted to CATransforms?

Reply:
Added.